spec/models/api_wrappers/kraken_api_wrapper_spec.rb

```diff
@@ -201,9 +289,25 @@ def stub_transactions(count: 1, price: 1.5, amount: 2.5)
   end
 
   it '#find_lost' do
-    stub_private_client
     stub_orders
 
-    described_class.orders.all? { |o| described_class.find_lost(o.type, o.price, o.amount).present? }
+    api_wrapper.orders.all? { |o| api_wrapper.find_lost(o.type, o.price, o.amount).present? }
+  end
+
+  it '#currency_pair' do
+    stub_assets
+    BitexBot::Settings.taker.kraken.currency_pair.should eq taker_settings.kraken.currency_pair
```

Comment:
Hmmm, this isn't really testing anything. It's comparing against it's own stub.

Reply:
In this case, we stub the output of a supposed reading of the settings, to test that the wrapper correctly map the data. What would be the correct way to test it?